### PR TITLE
Add project-name and env to slack notification

### LINKED ISSUE #1496: Add project name and environment to slack notification
**Is your feature request related to a problem? Please describe.**
Currently slack notification title looks like this:

If there are many dbt projects and environments we need to create lot of separate slack channels to distinguish from which dbt project and environment monitoring summary comes from. 

**Describe the solution you'd like**
To make it easier and more convenient we should add project name and environment to the header so it could look like:

- when --project-name is not provided
   
   
- when --project-name is provided()
   

**Would you be willing to contribute this feature?**
Yes, i will contribute and take care of this

### PR DESCRIPTION
**Pull request by kostek-pl**
For consistency with a report and for better visibility in slack notification, header is now extended by environment and project name(if provided via --project-name argument) like so:

- without --project-name
   image

- with --project-name
   image

Closes #1496

### CODE REVIEW

File: elementary/monitor/data_monitoring/report/slack_report_summary_message_builder.py [hunk 1]
```diff
@@ -34,9 +36,12 @@ def get_slack_message(
         )
         return super().get_slack_message()
 
-    def add_title_to_slack_alert(self):
+    def add_title_to_slack_alert(self, env: str, project_name: Optional[str] = None):
+        _project_name = f"for the {project_name} " if project_name else ""
```
Comment (ellakz) [inline elementary/monitor/data_monitoring/report/slack_report_summary_message_builder.py:40]:
```suggestion
        context = f"- {project_name} ({env})" if project_name else f"({env})"
```

File: elementary/monitor/data_monitoring/report/slack_report_summary_message_builder.py [hunk 2]
```diff
@@ -34,9 +36,12 @@ def get_slack_message(
...
         title_blocks = [
-            self.create_header_block(":mag: Monitoring summary"),
+            self.create_header_block(
+                f":mag: Monitoring summary {_project_name}({env})"
```
Comment (ellakz) [inline elementary/monitor/data_monitoring/report/slack_report_summary_message_builder.py:43]:
```suggestion
                f":mag: Monitoring summary {context}"
```

**Review by ellakz — COMMENTED** [review]
looks good! added 2 not-so-critical suggestions.

**Comment by haritamar** [comment]
Merged [here](https://github.com/elementary-data/elementary/pull/1506), so closing this PR

**Pull request closed without merging**